src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/resource/CalculationControllerTest.kt

```diff
@@ -365,7 +365,7 @@ class CalculationControllerTest {
       prisonerId,
       123456,
       LocalDateTime.now(),
-      UUID.randomUUID().toString(),
+      654321,
```

Comment:
Should this be a UUID string?

Reply:
was replacing UUID calc reference with Long calculation request id as I added the wrong one to the API